components/InputTypeCountry.js

```diff
@@ -97,6 +97,13 @@ class InputTypeCountry extends Component {
     );
   });
 
+  filterOptions(candidate, input) {
+    if (input) {
+      return candidate.data.country?.toLowerCase()?.includes(input);
```

Comment:
if the goal is to have a case insensitive search, why are you not lowercasing the input?